src/main/java/gov/cms/madie/models/measure/QdmMeasure.java

```diff
@@ -26,5 +26,5 @@ public class QdmMeasure extends Measure {
 	private boolean patientBasis = true;
 	private String rateAggregation;
 	private String improvementNotation;
-	private String improvementNotationOther;
+	private String improvementNotationDescription;
```

Comment:
Wouldn't we need this description for FHIR Measures ?

Reply:
Nvm FHIR has it at Group level, I just remembered

Reply:
yeah right